docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+# Configuration file to run ci tests in docker containers
```

Comment:
I would include the changes needed to use these files as part of the same commit. It is really quite complicated to figure out the interaction between these files or how will they be used in this commit.